### DISCUSSION TITLE
Suggestions: Collapse Function Definitions

### DISCUSSION THREAD

**Discussion by Weewooz** [JohnnyMorganz/luau-lsp#370] [Ideas]
I have a feature request I wanna make, right now with my small screen, it's really annoying when you can't collapse functions like in the attached image. Can someone add a feature where you can collapse function definitions?
 
 
![image](https://github.com/JohnnyMorganz/luau-lsp/assets/65644919/0c41f7b9-79d1-413d-aac9-27f1cd718f42)

**Comment by JohnnyMorganz** [comment]
Transferred this into an Issue https://github.com/JohnnyMorganz/luau-lsp/issues/371

**Comment by JohnnyMorganz** [comment]
This will be available in the next release, after https://github.com/JohnnyMorganz/luau-lsp/pull/378 is merged
